scripts/core/services/modalService.tsx

```diff
@@ -235,7 +235,7 @@ export default angular.module('superdesk.core.services.modal', [
         this.createCustomModal = function() {
             return new Promise((resolve) => {
                 $modal.open({
-                    template: '<div id="custom-modal-placeholder"></div>',
+                    template: '<div id="custom-modal-placeholder" data-test-id="custom-modal"></div>',
```

Comment:
Imagine you have a reusable button component. Would you make `data-test-id="custom-button"`? All buttons would be custom buttons then.

Take data-test-id as an argument to `createCustomModal`.